engine/_classes/class.bertatemplate.php

```diff
@@ -139,17 +140,23 @@ public function addContent($requestURI, $sectionName, &$sections, $tagName, &$ta
         $this->tagName = $tagName;
         $this->tags = &$tags;
 
+        $isEditMode = $this->environment == 'engine';
+
         // add entries...
         $this->content = &$content;
         $this->allContent = &$allContent;
 
+        $multisitesMenuRenderService = new MultisitesMenuRenderService(self::$options['MULTISITE'], $isEditMode, $this);
```

Comment:
Oh, no! Please do not use old Berta code instances, we are rewriting it in API. Get required data from API classes, see how it is realised in SectionEntryRenderService.